iOS/Flows/Conversation/Threads/ConversationThreadViewController.swift

```diff
@@ -137,7 +145,11 @@ class ConversationThreadViewController: DiffableCollectionViewController<Convers
     }
 
     override func getAnimationCycle() -> AnimationCycle? {
-        return nil
+        var cycle = super.getAnimationCycle()
+        cycle?.shouldConcatenate = false
+        cycle?.scrollToIndexPath = IndexPath(item: 0, section: 0)
+        cycle?.scrollPosition = .bottom
+        return cycle
```

Comment:
Should concatenate is false. This animates all the cells at the same time so the vertical line doesn't look weird.

Also scrolling to the first message which is the most recent one.